addons/wowl/static/src/components/dropdown/dropdown.scss

```diff
@@ -38,6 +38,9 @@ li.o_dropdown_item {
 
   > span {
     min-width: max-content;
+    a {
+      color: inherit;
+    }
```

Comment:
What do you think about following suggestion ?
I think it's better to be more generic about the inner style of the dropdown.

```suggestion
```

And add `color: inherit;` before line 36:
```css
 * {
    color: inherit;
    line-height: inherit;
  }
```

Reply:
Ok